js/ui/gallery.js

```diff
@@ -638,25 +641,16 @@ const Gallery = CollectionWidget.inherit({
             this._releaseInvisibleItems();
             return;
         }
-
-        this._itemElements().each((function(index, item) {
-            if(this.option('selectedIndex') === index) {
-                $(item).removeClass(GALLERY_INVISIBLE_ITEM_CLASS);
-            } else {
-                $(item).addClass(GALLERY_INVISIBLE_ITEM_CLASS);
+        const selectedIndex = this.option('selectedIndex');
+        this._itemElements().each((index, item) => {
+            if(selectedIndex !== index) {
+                $(item).find(ITEM_CONTENT_SELECTOR).addClass(GALLERY_INVISIBLE_ITEM_CLASS);
             }
-        }).bind(this));
-
-        this._getLoopedItems()
-            .addClass(GALLERY_INVISIBLE_ITEM_CLASS);
+        });
     },
 
     _releaseInvisibleItems: function() {
-        this._itemElements()
-            .removeClass(GALLERY_INVISIBLE_ITEM_CLASS);
-
-        this._getLoopedItems()
-            .removeClass(GALLERY_INVISIBLE_ITEM_CLASS);
+        this._itemElements().find(ITEM_CONTENT_SELECTOR).removeClass(GALLERY_INVISIBLE_ITEM_CLASS);
```

Comment:
what about looped items?

Reply:
The '_itemElements' returns all elements